mk/external.mk

```diff
@@ -34,14 +34,23 @@ EXTERNAL_SRC = LINUX BUSYBOX
 $(foreach T,$(EXTERNAL_SRC),$(eval $(download-n-extract)))
 
 # Build Linux kernel image
-LINUX_IMG = $(OUT)/bzImage
+ifeq ($(ARCH), x86_64)
+LINUX_IMG_NAME = bzImage
+LINUX_ARCH = x86
+endif
+ifeq ($(ARCH), aarch64)
```

Comment:
Replace this line with `else ifeq ($(ARCH), aarch64)`.
In addition, append the following:
```
else
    $(error Unsupported architecture)
endif
```